assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -272,6 +272,7 @@ db.jndi.datasource.name=java:/comp/env/jdbc/che
 
 # OpenShift related properties
 che.openshift.endpoint=https://192.168.64.2:8443/
+che.openshift.token=
 che.openshift.username=openshift-dev
 che.openshift.password=devel
 che.openshift.project=eclipse-che
```

Comment:
properties that star by che.* can be overrided by ENV variables

che.openshift.token --> CHE_OPENSHIFT_TOKEN=""